cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -174,6 +174,20 @@ def get_custom_tags(config: BaseClusterConfig, raw_dict: bool = False):
         custom_tags = {tag.key: tag.value for tag in config.tags} if config.tags else {}
     else:
         custom_tags = [CfnTag(key=tag.key, value=tag.value) for tag in config.tags] if config.tags else []
+
+    if config.scheduling.scheduler == "plugin":
+        scheduler_plugin_tags = get_attr(config, "scheduling.settings.scheduler_definition.tags")
+        if raw_dict:
+            custom_scheduler_plugin_tags = (
+                {tag.key: tag.value for tag in scheduler_plugin_tags} if scheduler_plugin_tags else {}
+            )
+            custom_tags.update(custom_scheduler_plugin_tags)
+        else:
+            custom_scheduler_plugin_tags = (
+                [CfnTag(key=tag.key, value=tag.value) for tag in scheduler_plugin_tags] if scheduler_plugin_tags else []
+            )
+            custom_tags += custom_scheduler_plugin_tags
+
```

Comment:
can we get rid of this custom logic and use `config.get_cluster_tags()` instead of config.tags in the existing code above?

Reply:
Good idea! Changed